arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java

```diff
@@ -296,6 +304,218 @@ void testCall(TestInfo testInfo) {
     toyExecutionEnvironmentV2.run();
   }
 
+  @Tag("nightly")
+  @ParameterizedTest
+  @MethodSource({"testMxpxThresholdSource"})
+  void testMxpxThreshold(
+      OpCode opCode,
+      BigInteger offset1,
+      BigInteger offset2,
+      BigInteger size1,
+      BigInteger size2,
+      TestInfo testInfo) {
+    BytecodeCompiler program = BytecodeCompiler.newProgram(chainConfig);
+
+    switch (opCode) {
+        // 1 offset
+      case MSTORE, MSTORE8 -> program
+          .push(0) // value
+          .push(offset1)
+          .op(opCode);
+        // 2 offsets, 1 size
+      case CODECOPY -> program.push(size1).push(offset2).push(offset1).op(opCode);
```

Comment:
This isn't a 2 offset 1 size opcode from the `MXP` point of view: `MCOPY` is the only such opcode.